db/migrations/versions/c20eef2754cf_.py

```diff
@@ -0,0 +1,110 @@
+"""empty message
+
+Revision ID: c20eef2754cf
+Revises: 91ed555648be
+Create Date: 2022-07-18 16:52:27.694556
+
+"""
+import sqlalchemy as sa
+import sqlalchemy_utils
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "c20eef2754cf"
+down_revision = "91ed555648be"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "criteria",
+        sa.Column(
+            "criteria_id",
+            sqlalchemy_utils.types.uuid.UUIDType(binary=False),
+            nullable=False,
+        ),
+        sa.Column("criteria_name", sa.Text(), nullable=True),
```

Comment:
Can these be just `id` and `name`? (as the criteria context is derivable from the table name.)